code/game/machinery/PDApainter.dm

```diff
@@ -334,8 +334,13 @@
 				stored_pda.icon = initial(pda_path.icon)
 			stored_pda.icon_state = initial(pda_path.icon_state)
 			stored_pda.desc = initial(pda_path.desc)
+			stored_pda.reset_imprint()
 
 			return TRUE
+
+		if("reset_pda")
+			stored_pda.reset_imprint()
```

Comment:
Does this button exist?

Reply:
not implemented yet, probably gonna scrap

Reply:
I think it would be a good idea, but maybe it should also reset the paint to assistant. would let heads of staff be able to demote their PDAs as well.

Reply:
> would let heads of staff be able to demote their PDAs as well

I was intending to add this as a way to "retire" PDAs if you ever need to, because resetting imprints removes the PDA's messenger from the list of available messengers. Demoting PDAs can be done by changing the color of PDAs back to normal, as well as deleting all apps manually.

Reply:
> I was intending to add this as a way to "retire" PDAs if you ever need to, because resetting imprints removes the PDA's messenger from the list of available messengers. Demoting PDAs can be done by changing the color of PDAs back to normal, as well as deleting all apps manually.

This would also be handy for, say, space explorers when finding the clear PDA - currently the only realistic way to get the old one off the messenger is to destroy it somehow. (If turning send/receive off is intended do to that, well, it doesn't work unfortunately).

Not that it should necessarily be your responsibility, but if you do feel like not scrapping that afterall...

Reply:
Only the ones that have a name associated with an ID get displayed on the messenger list. The ones marked as `invisible || monitor_hidden` are filtered out